index.html

```diff
@@ -312,12 +312,14 @@ <h4 class="caption">Frameworks</h4>
 
         <ul class="tools tabs-bottom-line">
           <li class="tabs">
-            <h4 class="caption">Skills</h4>
+            <h4 class="caption">Skills </h4> 
+            <a href="#">
             <img
               class="left-sidebar"
               alt="open-tab"
               src="Images/Disabled.png"
             />
+          </a>
```

Comment:
We added on line number 316  <a> section .to make the Skills arrow have Tab interaction.